lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -10,18 +24,16 @@
 
```

Comment:
Not likely anything we can do about this, but in Safari (blind Mac users can only use Safari, any other browser does not work with Voiceover) I get the Flash version of the camera control and it's not accessible.  It's impossible for me to enable the camera.  I can't move focus to the control.